common/scripts/Ubuntu_14.04/installVault.sh

```diff
@@ -53,9 +53,17 @@ check_vault() {
 }
 
 main() {
+  check_vault=""
   {
-    type vault &> /dev/null && echo "Vault already installed, skipping" && return
-  }
+    type vault &> /dev/null
```

Comment:
i dont think we run vault & gitlab as upstart. `service --status-all 2>&1 | grep vault` doesn't return anything.